manifests/m/Microsoft/Sysinternals/2024-12-16/Microsoft.Sysinternals.installer.yaml

```diff
@@ -0,0 +1,165 @@
+- RelativeFilePath: adrestore64.exe
+- RelativeFilePath: autorunsc.exe
+- RelativeFilePath: autorunsc64.exe
+- RelativeFilePath: dbgview64.exe
+- RelativeFilePath: disk2vhd.exe
+- RelativeFilePath: disk2vhd64.exe
+- RelativeFilePath: diskext.exe
+- RelativeFilePath: diskext64.exe
+- RelativeFilePath: du.exe
+- RelativeFilePath: du64.exe
+- RelativeFilePath: efsdump.exe
+- RelativeFilePath: handle.exe
+- RelativeFilePath: handle64.exe
+- RelativeFilePath: hex2dec.exe
+- RelativeFilePath: hex2dec64.exe
+- RelativeFilePath: junction.exe
+- RelativeFilePath: junction64.exe
+- RelativeFilePath: ldmdump.exe
+- RelativeFilePath: livekd.exe
+- RelativeFilePath: livekd64.exe
+- RelativeFilePath: logonsessions.exe
+- RelativeFilePath: logonsessions64.exe
+- RelativeFilePath: movefile.exe
+- RelativeFilePath: movefile64.exe
+- RelativeFilePath: notmyfault.exe
+- RelativeFilePath: notmyfault64.exe
+- RelativeFilePath: notmyfaultc.exe
+- RelativeFilePath: notmyfaultc64.exe
+- RelativeFilePath: ntfsinfo.exe
+- RelativeFilePath: ntfsinfo64.exe
+- RelativeFilePath: pendmoves.exe
+- RelativeFilePath: pendmoves64.exe
+- RelativeFilePath: pipelist.exe
+- RelativeFilePath: pipelist64.exe
+- RelativeFilePath: portmon.exe
+- RelativeFilePath: procdump.exe
+- RelativeFilePath: procdump64.exe
+- RelativeFilePath: procexp.exe
+- RelativeFilePath: procexp64.exe
+- RelativeFilePath: psfile.exe
+- RelativeFilePath: psfile64.exe
+- RelativeFilePath: pskill.exe
+- RelativeFilePath: pskill64.exe
+- RelativeFilePath: pslist.exe
+- RelativeFilePath: pslist64.exe
+- RelativeFilePath: psloglist.exe
+- RelativeFilePath: psloglist64.exe
+- RelativeFilePath: pspasswd.exe
+- RelativeFilePath: pspasswd64.exe
+- RelativeFilePath: psping.exe
+- RelativeFilePath: psping64.exe
+- RelativeFilePath: psshutdown.exe
+- RelativeFilePath: psshutdown64.exe
+- RelativeFilePath: pssuspend.exe
+- RelativeFilePath: pssuspend64.exe
+- RelativeFilePath: regjump.exe
+- RelativeFilePath: ru.exe
+- RelativeFilePath: ru64.exe
+- RelativeFilePath: sdelete.exe
+- RelativeFilePath: sdelete64.exe
+- RelativeFilePath: sigcheck.exe
+- RelativeFilePath: sigcheck64.exe
+- RelativeFilePath: streams.exe
+- RelativeFilePath: streams64.exe
+- RelativeFilePath: strings.exe
+- RelativeFilePath: strings64.exe
+- RelativeFilePath: sync.exe
+- RelativeFilePath: sync64.exe
+- RelativeFilePath: tcpvcon.exe
+- RelativeFilePath: tcpvcon64.exe
+- RelativeFilePath: tcpview.exe
+- RelativeFilePath: tcpview64.exe
+- RelativeFilePath: vmmap.exe
+- RelativeFilePath: vmmap64.exe
+- RelativeFilePath: whois.exe
+- RelativeFilePath: whois64.exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-12-16
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://download.sysinternals.com/files/SysinternalsSuite.zip
+  InstallerSha256: C6C6FC3A965A8E4BC0943EE4586CCE86789913DDD25E808BFB61426776253863
+ManifestType: installer
+ManifestVersion: 1.9.0

```

Comment:
Why is the manifest version bumped up by 3 minor versions? From 1.6.0 to 1.9.0?

Reply:
Please resolve this comment. It is intended as a question only.

Reply:
It was done automatically by Komac, I thought it a bit odd too considering the other manifest was generated the same way.